.travis.yml

```diff
@@ -0,0 +1,19 @@
+sudo: false
+language: python
+python:
+- '3.4'
+- '3.5'
+install:
+- pip install pymodbus3 pyserial
```

Comment:
As you commented pymodbus3 is only for python3. There is a `py3` branch with python3 compatibile version, while `master` works only with python2. 
So change this PR to be merged towards `py3`, not `master` and in some free time I will make it work with both python versions